server/services/Mailer.js

```diff
@@ -34,21 +32,18 @@ class Mailer extends helper.Mail {
   addRecipients() {
     const personalize = new helper.Personalization();
 
-    this.recipients.forEach(recipient => {
-      personalize.addTo(recipient);
-    });
+    this.recipients.forEach(recipient => personalize.addTo(recipient));
     this.addPersonalization(personalize);
   }
 
-  async send() {
+  send() {
     const request = this.sgApi.emptyRequest({
       method: 'POST',
       path: '/v3/mail/send',
       body: this.toJSON()
     });
 
-    const response = await this.sgApi.API(request);
-    return response;
+    return this.sgApi.API(request);
```

Comment:
This is incorrect usage of async/await. It's being used here, and in the function that calls this method. If it's used here, this function does not return a promise, simply the result of the `sgApi.API` call.

Since survey routes calls this and other async code, it makes the most sense to remove async/await here, and just have this return the promise, then use async/await int he calling method (which is in surveyRoutes.js and looks like this)

```
try {
      await mailer.send(); //with the old code this line was wrong, as you don't "await" a response object
      await survey.save();
      req.user.credits -= 1;
      const user = await req.user.save();

      res.send(user);
    } catch (err) {
      res.status(422).send(err);
    }
```